system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -197,9 +197,64 @@ const TopSites = props => (<section>
       link={link}
       index={index} />)}
   </ul>
+  <TopSitesEditIntl {...props} />
 </section>);
 
+class TopSitesEdit extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {showEditModal: false};
+    this.onEditButtonClick = this.onEditButtonClick.bind(this);
+  }
+  getDefaultProps() { return {dispatch: () => {}}; }
```

Comment:
oh right, this is really needed since the component doesn't even dispatch anything *yet*. nice catch